core/src/com/etheller/warsmash/parsers/jass/Jass2.java

```diff
@@ -1244,3914 +1039,2266 @@ public boolean call(final CUnit unit) {
-					final String environmentName = arguments.get(0).visit(StringJassValueVisitor.getInstance());
-					System.err.println("#########");
-					System.err.println("# Engine requested sound environment: " + environmentName);
-					System.err.println("# I don't know how to do that on LibGDX, so for now, I do nothing!");
-					System.err.println("#########");
-					return null;
-				}
+			jassProgramVisitor.getJassNativeManager().createNative("GetIssuedOrderId", (arguments, globalScope, triggerScope) -> {
+				return new IntegerJassValue(((CommonTriggerExecutionScope) triggerScope).getIssuedOrderId());
 			});
-			jassProgramVisitor.getJassNativeManager().createNative("CreateMIDISound", new JassFunction() {
-				@Override
-				public JassValue call(final List<JassValue> arguments, final GlobalScope globalScope,
-						final TriggerExecutionScope triggerScope) {
-					final String soundLabel = arguments.get(0).visit(StringJassValueVisitor.getInstance());
-					final int fadeInRate = arguments.get(1).visit(IntegerJassValueVisitor.getInstance());
-					final int fadeOutRate = arguments.get(2).visit(IntegerJassValueVisitor.getInstance());
-					return new HandleJassValue(soundType, new CMIDISound(soundLabel, fadeInRate, fadeOutRate));
-				}
+			jassProgramVisitor.getJassNativeManager().createNative("GetOrderPointX", (arguments, globalScope, triggerScope) -> {
+				return new RealJassValue(((CommonTriggerExecutionScope) triggerScope).getOrderPointX());
 			});
-			jassProgramVisitor.getJassNativeManager().createNative("GetFloatGameState", new JassFunction() {
-				@Override
-				public JassValue call(final List<JassValue> arguments, final GlobalScope globalScope,
-						final TriggerExecutionScope triggerScope) {
-					final CGameState gameState = arguments.get(0).visit(ObjectJassValueVisitor.getInstance());
-					switch (gameState) {
-					case TIME_OF_DAY:
-						return new RealJassValue(CommonEnvironment.this.simulation.getGameTimeOfDay());
-					}
-					throw new IllegalArgumentException("Not a float game state: " + gameState);
-				}
+			jassProgramVisitor.getJassNativeManager().createNative("GetOrderPointY", (arguments, globalScope, triggerScope) -> {
+				return new RealJassValue(((CommonTriggerExecutionScope) triggerScope).getOrderPointY());
 			});
-			jassProgramVisitor.getJassNativeManager().createNative("GetIntegerGameState", new JassFunction() {
-				@Override
-				public JassValue call(final List<JassValue> arguments, final GlobalScope globalScope,
-						final TriggerExecutionScope triggerScope) {
-					final CGameState gameState = arguments.get(0).visit(ObjectJassValueVisitor.getInstance());
-					switch (gameState) {
-					case DISCONNECTED:
-					case DIVINE_INTERVENTION:
-						return new IntegerJassValue(0); // TODO
-					}
-					throw new IllegalArgumentException("Not an integer game state: " + gameState);
-				}
+			jassProgramVisitor.getJassNativeManager().createNative("GetOrderPointLc", (arguments, globalScope, triggerScope) -> {
```

Comment:
This function name is mistyped and says `GetOrderPointLc` instead of `GetOrderPointLoc`